### PR TITLE
Updated Docs + Error Handling Page

### PR DESCRIPTION
**Pull request by Vardan2009**

### CODE REVIEW

**Comment by Almas-Ali** [comment]
Can you please resolve those conflicts?

**Comment by Almas-Ali** [comment]
Please check in the `radon-project:master` `docs/exceptions.md` and combine this with your `docs/error-handling.md`.

**Comment by Vardan2009** [comment]
I think "Error Handling" is a better title for the page

File: docs/exceptions.md [hunk 1]
```diff
@@ -71,6 +72,43 @@ ValueError: 2 + 2 != 4
...
+    {
+        x = 10 / y
+    }
+    catch as e
+    {
+        raise FunctionError("risky_operation", e)
+    } 
+}
+
+risky_operation(0)
```
Comment (Almas-Ali) [inline docs/exceptions.md]: Please test your code before putting on public docs.

Also follow K&R formatting for writing code in docs.

**Review by Almas-Ali — CHANGES_REQUESTED** [review]

**Review by Almas-Ali — APPROVED** [review]

**Pull request merged**